README.md

```diff
@@ -217,6 +217,7 @@ A curated list of awesome things related to <a href="//docsify.js.org">docsify</
 - [docsify-chat](https://github.com/dcyuki/docsify-chat) - A docsify plugin for generate chat panel from markdown.
 - [docsify-pagination-keyboard-helper](https://github.com/Brannua/docsify-pagination-keyboard-helper) - A plugin for docsify site that enables you to turn pages quickly by using keyboard shortcuts.
 - [docsify-plugin-runkit](https://jhildenbiddle.github.io/docsify-plugin-runkit) - A docsify plugin for rendering and embedding interactive JavaScript REPLs powered by [RunKit](https://runkit.com/home) [@jhildenbiddle](https://github.com/jhildenbiddle/)
+- [WPDocsify](https://github.com/mitchell-b-chelin/WPDocsify) - A magical documentation library for Wordpress.  
```

Comment:
This is a WordPress plugin, not a docsify plugin.

I think it can be put in `Community Resources

Reply:
Nice WP integration! Yeah, I think it is better in the community resources section.